benchmark_runner/workloads/bootstorm_vm.py

```diff
@@ -236,87 +236,86 @@ def _verify_single_vm_access_wrapper(self, vm_name, return_dict):
         self._oc.save_describe_yml(vm_name, str(vm_access).lower() == 'true', output_dir=self._run_artifacts_path)
         return_dict[vm_name] = vm_access
 
-    def _verify_vms_access_in_parallel(self, vm_names):
+    def _verify_vms_access_in_parallel(self, vm_names: list):
         """
-        This method verifies VM access in parallel
-        :param vm_names:
-        :return:
+        Verifies access to a list of VMs in parallel using subprocesses.
+
+        :param vm_names: List of VM names to verify.
+        :return: List of VM names that failed the access verification.
         """
-        failure_vms = []
-        manager = Manager()
-        return_dict = manager.dict()
+        if vm_names:
```

Comment:
Need to return None if there are no VM names.

Reply:
Done,
```
if not vm_names:
    return []
```